tests/foreman/api/test_repository.py

```diff
@@ -394,6 +394,37 @@ def test_positive_create_same_name_different_orgs(self):
         repo2 = entities.Repository(name=repo1.name).create()
         self.assertEqual(repo1.name, repo2.name)
 
+    @tier2
+    def test_positive_create_puppet_repo_same_url_different_orgs(self):
+        """Create two repos with the same URL in two different organizations.
+
+        @id: 7c74c2b8-732a-4c47-8ad9-697121db05be
+
+        @Assert: Repositories are created and puppet modules are visible from
+        different organizations.
+
+        @CaseLevel: Integration
+        """
+        url = 'https://omaciel.fedorapeople.org/7c74c2b8/'
```

Comment:
According to issue description it was requested that URL should be unique and not used by any other tests. So it is better to keep URL inside test itself, otherwise someone can use that URL.
@sghai can you please provide more information?